R/nhs_footer.R

```diff
@@ -1,7 +1,20 @@
-#' nhs_footer Function
+#' nhs_footer
 #'
-#' @importFrom shiny tagList
-nhs_footer <- function() {
+#' NHS style footer element
+#'
+#' @param a11y_statement_url URL of accessibility statement
+#' @param email Contact email, by default 'nhsbsa.dall@nhs.net'
+#' @param github_url URL of app GitHub repo, by default 
+#'   https://github.com/nhsbsa-data-analytics
+#' 
+#' @return HTML
+#' @export
+#'
+#' @examples
+#' nhs_footer()
+nhs_footer <- function(a11y_statement_url = "#",
+                       email = "nhsbsa.dall@nhs.net",
+                       github_url = "https://github.com/nhsbsa-data-analytics") {
```

Comment:
Added some args to pass in variable components. Plus document as is exported.

Reply:
same quick note on dall@nhsbsa.nhs.uk

Reply:
Sorted